### DISCUSSION TITLE
Error: Error building site: TOCSS: failed to transform "scss/style.scss" (text/x-scss)

### DISCUSSION THREAD

**Discussion by yezihack** [CaiJimmy/hugo-theme-stack#639] [Q&A]
hugo v0.101.0-466fa43c16709b4483689930a4f9ac8add5c9f66 windows/amd64 BuildDate=2022-06-16T07:09:16Z VendorInfo=gohugoio
go version go1.18.3 windows/amd64
```
PS E:\mysite> hugo server
Start building sites …
hugo v0.101.0-466fa43c16709b4483689930a4f9ac8add5c9f66 windows/amd64 BuildDate=2022-06-16T07:09:16Z VendorInfo=gohugoio
WARN 2022/07/06 23:37:57 Search page not found. Create a page with layout: search.
WARN 2022/07/06 23:37:57 Archives page not found. Create a page with layout: archives.
Error: Error building site: TOCSS: failed to transform "scss/style.scss" (text/x-scss). Check your Hugo installation; you need the extended version to build SCSS/SASS.: this feature is not available in your current Hugo version, see https://goo.gl/YMrWcn for more information
```

unable to start hugo

**Comment by CaiJimmy** [comment]
"Error: Error building site: TOCSS: failed to transform "scss/style.scss" (text/x-scss). Check your Hugo installation; you need the **extended** version to build SCSS/SASS.: **this feature is not available in your current Hugo version**, see https://goo.gl/YMrWcn for more information"

Use Hugo extended version.

**Comment by Mryuksel101** [comment]
> hugo v0.101.0-466fa43c16709b4483689930a4f9ac8add5c9f66 windows/amd64 BuildDate=2022-06-16T07:09:16Z VendorInfo=gohugoio go version go1.18.3 windows/amd64
> 
> ```
> PS E:\mysite> hugo server
> Start building sites …
> hugo v0.101.0-466fa43c16709b4483689930a4f9ac8add5c9f66 windows/amd64 BuildDate=2022-06-16T07:09:16Z VendorInfo=gohugoio
> WARN 2022/07/06 23:37:57 Search page not found. Create a page with layout: search.
> WARN 2022/07/06 23:37:57 Archives page not found. Create a page with layout: archives.
> Error: Error building site: TOCSS: failed to transform "scss/style.scss" (text/x-scss). Check your Hugo installation; you need the extended version to build SCSS/SASS.: this feature is not available in your current Hugo version, see https://goo.gl/YMrWcn for more information
> ```
> 
> unable to start hugo
![Ekran Görüntüsü (575)](https://user-images.githubusercontent.com/99680265/178472120-5b8c0523-ce3d-4bac-8022-40bdbc036282.png)

extended must be true
